source/draw/gpu/opengl/OpenGLFence.ooc

```diff
@@ -17,19 +17,37 @@
 use ooc-draw-gpu
 import backend/[GLFence, GLContext]
 import OpenGLContext
+import threading/[Mutex, WaitCondition]
 
 version(!gpuOff) {
 OpenGLFence: class extends GpuFence {
 	_backend: GLFence
-	init: func (context: OpenGLContext) {
-		this _backend = context backend as GLContext createFence()
-	}
+	_syncCondition := WaitCondition new()
+	_mutex := Mutex new()
```

Comment:
I like `WaitLock` but it may be too close to `WaitCondition`. `SynchronizationLock` is also good.